README.md

```diff
@@ -1,2 +1,27 @@
-# agartex-app
-Web application for AgarTeX
+# AgarTeX App
+
+This repository contains AgarTeX frontend application code.
+
+## Development
+
+```
+nvm use         # sets node version to specified in .nvmrc file
+npm install     # installs all dependencies from package_lock.json
+npm run start   # start development server on localhost:3000
+```
+
+## Linter and testing
+
+```
+npm run lint    # runs linter check
+npm run test    # runs jest tests
+```
+
+## Build 
+
+```
+npm run build               # creates production build in 'build' folder
+node scripts/run-server.js  # runs production server on localhost:5000
```

Comment:
Why dev and prod server have different port?

Reply:
i believe port 3000 for dev is a react convention, so I'd leave prod on different port for clarity. But really, it doesn't matter that much